Makefile

```diff
@@ -56,13 +68,14 @@ ifeq ($(PROJECT_INSTALL), config)
 else
 	$(call php, drush si $(PROFILE_NAME) --db-url=$(DB_URL) --account-pass=admin -y --site-name="$(SITE_NAME)" --site-mail="$(SITE_MAIL)" install_configure_form.site_default_country=FR install_configure_form.date_default_timezone=Europe/Paris)
 endif
-	#$(call php, drush en $(MODULES) -y)
-	#$(call php, drush pmu $(MODULES) -y)
+ifneq ($(strip $(MODULES)),)
+	$(call php, drush en $(MODULES) -y)
```

Comment:
would be great to document the purpose of that - default_content & co

Reply:
Documenting in progress